client/txn_test.go

```diff
@@ -498,8 +498,12 @@ func TestAbortTransactionOnCommitErrors(t *testing.T) {
 		}, nil))
 
 		txn := NewTxn(*db)
-		_ = txn.Put("a", "b")
-		_ = txn.Commit()
+		if pErr := txn.Put("a", "b"); pErr != nil {
+			t.Errorf("put failed: %s", pErr)
```

Comment:
These should generally be `Fatalf` instead of `Errorf`. If the put fails it doesn't make sense to proceed to the commit, and if continuing causes something to panic the underlying `Errorf` message will be lost.